src/context/User.Context.js

```diff
@@ -53,6 +54,53 @@ const UserProvider = ({children}) => {
     return temp;
   };
 
+  const fetchExercises = async () => {
+    // create temp muscle and equipment arr to use in api call to request filtered results
+    let tempMuscleArr = [];
+    for (let i = 0; i < selectedMuscles.length; i++) {
+      const [filteredMuscleGroup] = muscleGroup.filter(
+        muscle => muscle.name_en === selectedMuscles[i],
+      );
+      tempMuscleArr.push(filteredMuscleGroup.id);
+    }
+    const muscleIds = tempMuscleArr.join(',');
+
+    let tempEquipmentArr = [];
+    for (let i = 0; i < selectedEquipment.length; i++) {
+      const [filteredEquipmentTypes] = equipmentTypes.filter(
+        equipment => equipment.name === selectedEquipment[i],
+      );
+      tempEquipmentArr.push(filteredEquipmentTypes.id);
+    }
+    const equipmentIds = tempEquipmentArr.join(',');
+
```

Comment:
You’re doing the same thing twice right? Can you create a function here so you can just call it with different params